tests/contrib/test_heroku.py

```diff
@@ -152,6 +152,7 @@ def _check_heroku_deployment(meta):
     assert len(res) == 1
 
 
+@flaky
```

Comment:
please make sure that it retries by default and fails if no attempts are successful. We should not keep tests that we don't check results for.

Reply:
Checked, this works as expected.